homeassistant/helpers/selector.py

```diff
@@ -259,10 +262,12 @@ class TimeSelector(Selector):
 
     selector_type = "time"
 
-    CONFIG_SCHEMA = vol.Schema({})
+    CONFIG_SCHEMA = vol.Schema({vol.Optional("as_str", default=False): bool})
```

Comment:
Maybe it should be an option for all selectors where the data from frontend is not modified during validation?

Reply:
Nah, it's happening only for a few I think (time/date/datetime). Bit mixed how I feel about this. I wonder if it should always be a string?